### ISSUE TITLE
The file you tried to load or one of its dependencies is probably missing

### ISSUE THREAD

**Issue by broprosam** [staxrip/staxrip#303]
on StaxRip x64 2.1.3.0, using VapourSynth filter, whenever I try to turn on the MCTemporalDenoise, I'm facing this Script Error Popup. Please tell me how to fix that.

> Script Error
> 
> Python exception: Failed to load C:\StaxRip-x64-2.1.3.0-stable\Apps\Plugins\VS\EEDI3m\EEDI3m.dll. GetLastError() returned 126. The file you tried to load or one of its dependencies is probably missing.
> 
> Traceback (most recent call last):
>   File "src\cython\vapoursynth.pyx", line 1956, in vapoursynth.vpy_evaluateScript
>   File "src\cython\vapoursynth.pyx", line 1957, in vapoursynth.vpy_evaluateScript
>   File "C:\Users\AAG4liB\Downloads\Video_temp\Video_new_preview.vpy", line 16, in <module>
>     core.std.LoadPlugin(r"C:\StaxRip-x64-2.1.3.0-stable\Apps\Plugins\VS\EEDI3m\EEDI3m.dll", altsearchpath=True)
>   File "src\cython\vapoursynth.pyx", line 1862, in vapoursynth.Function.__call__
> vapoursynth.Error: Failed to load C:\StaxRip-x64-2.1.3.0-stable\Apps\Plugins\VS\EEDI3m\EEDI3m.dll. GetLastError() returned 126. The file you tried to load or one of its dependencies is probably missing.

**Comment by almanteka** [comment]
at the top go to: Apps > Manage > left size select VapourSynth > EEDI3m > status OK?
![EED13m](https://user-images.githubusercontent.com/42816936/90797044-191a6a00-e2de-11ea-9110-f726b1ff7b40.jpg)

**Comment by broprosam** [comment]
> at the top go to: Apps > Manage > left size select VapourSynth > EEDI3m > status OK?
> ![EED13m](https://user-images.githubusercontent.com/42816936/90797044-191a6a00-e2de-11ea-9110-f726b1ff7b40.jpg)

Yes. The Status is OK
![Screenshot](https://user-images.githubusercontent.com/69974487/90800467-8d80f780-e336-11ea-9ed6-99c2a46a75ba.png)

**Comment by katyurua** [comment]
![2020-10-24_113118](https://user-images.githubusercontent.com/59998095/97067076-8901e500-15ec-11eb-8ea4-928bfce3b5e9.jpg)

**I have this problem too!** 

both beta and stable Ver.2130

**### I only have this problem on xeon platform, no problems on i5 4xxx platform**

Please tell me how to fix that

![2020-10-24_113615](https://user-images.githubusercontent.com/59998095/97067148-4bea2280-15ed-11eb-8ee6-1e2c57565248.jpg)


![2020-10-24_113418](https://user-images.githubusercontent.com/59998095/97067102-e85ff500-15ec-11eb-85ee-c550048e88f5.jpg)

**Comment by stax76** [comment]
The only thing you can do is finding a DLL that is compatible with your CPU or use another filter not using this DLL.

**Comment by katyurua** [comment]
> The only thing you can do is finding a DLL that is compatible with your CPU or use another filter not using this DLL.

I fixed this by install Intel opencl runtime for Xeon processor

[https://software.intel.com/content/www/us/en/develop/articles/opencl-drivers.html](url)

**Comment by stax76** [comment]
Good to hear that it works now.

**Comment by BlohoJo** [comment]
I am also having this problem.

The problem is only on a Windows 10 machine. Windows 7 machine works fine.

Both machines are GPU-less, and have the above Intel OpenCL CPU Runtime installed.

I am at a loss as to why it works on Windows 7, but not Windows 10.

**Comment by BlohoJo** [comment]
Looks like I figured out the problem... for some reason, even though I had installed the above OpenCL CPU runtime, the `OpenCL.dll` files did not get copied into `C:\Windows\System32`, and `C:\Windows\SysWOW64`. I used Universal Extractor on the .msi installer. `System64Folder\OpenCL.dll` gets copied into `C:\Windows\System32`, and `SystemFolder\OpenCL.dll` gets copied into `C:\Windows\SysWOW64`.

Not sure if the Intel CPU runtime for OpenCL is fully compatible / supported on Windows 10... but it seems to work now...
